app/src/lib/feature-flag.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Enables the application to opt-in for preview features based on runtime
+ * checks. This is backed by the GITHUB_DESKTOP_PREVIEW_FEATURES environment
+ * variable, which is checked for non-development environments.
+ */
+export function enablePreviewFeatures(): boolean {
```

Comment:
This could just be `return __DEV__ || process.env.GITHUB_DESKTOP_PREVIEW_FEATURES`.

Reply:
Perhaps, but I don't want to lock down these rules too much too early. I was thinking of checking for `GITHUB_DESKTOP_PREVIEW_FEATURES` first and overriding whatever your release channel was, for example.